### PR TITLE
Add blmgroep.nl as Private Domain

### PR DESCRIPTION
**Pull request by githeuvel**
* [x] Description of Organization
* [x] Reason for PSL Inclusion
* [x] DNS verification via dig
* [x] Run Syntax Checker (make test)

Description of Organization
====

Organization Website: no website

This domain is used for private and personal(/student) development. 

Reason for PSL Inclusion
====

As a private teacher I provide free subdomains to my students. This PR will help them with cookie domain scoping.

DNS Verification via dig
=======

```
dig +short TXT _psl.blmgroep.nl
"https://github.com/publicsuffix/list/pull/876"
```

make test
=========

Ran the test, all passed.

### CODE REVIEW

**Comment by sleevi** [comment]
Can you provide more details about this request? It's unclear how Cloudflare relates to adding to the PSL, or the securing services desire.

**Comment by dnsguru** [comment]
@githeuvel please reply to @sleevi question above about cloudflare for consideration of this request, or close this.  We shall close this if we have not heard a response by April 20, 2020

**Comment by githeuvel** [comment]
@dnsguru @sleevi , sorry for the late reply.

The relation between adding to the PSL and device security is an indirect relation.

1. The sub-domain is meant for students developing web applications.
1. To protect these applications, we would like to use [Cloudflare](https://www.cloudflare.com/learning/security/what-is-web-application-security/), which has a free plan for students ("hobby projects")
1. Cloudflare only supports top-level domains (or actually first level beneath top-level) on their "student" (free) plan.

So in order to provide the students a fairly protected environment, using Cloudflare, we need to fullfill the Cloudflare. 

In this specific case `blmgroep.nl` can be used. But to provide each students with (e.g.) `student1.blmgroep.nl` (etc). we need this change.

This is just the first step in possibly more problems students may run into; ranging from "Enterprise" (not available for students), to security related (cookies).

This request is similar to:
https://github.com/publicsuffix/list/pull/999
https://github.com/publicsuffix/list/pull/960

**Comment by sleevi** [comment]
Again, I'm not sure what "protected" and "Cloudflare" have to do, so I'm hoping you can clarify.

**Comment by dnsguru** [comment]
@sleevi just like in #990 it seems as though users seek to add subdomains into cloudflare's system and there is a gating imposed on having these present in PSL.

**Comment by sleevi** [comment]
@dnsguru That doesn't answer my question though. I'm trying to discern how this is different than Let's Encrypt rate limits. Saying "security" isn't enough to help us figure out.

**Comment by dnsguru** [comment]
[See Step 2 of this page from cloudflare's support pages](https://support.cloudflare.com/hc/en-us/articles/205359838-I-cannot-add-my-domain-to-Cloudflare-)

**Comment by sleevi** [comment]
That doesn't answer my question.

**Comment by dnsguru** [comment]
> That doesn't answer my question.

I poked around on cloudflare and found the article link above, which is related to newTLDs that may not yet have been in the list or ccTLDs that are not listed.   It appears that cloudflare will allow a "registerable" domain to be added to an account, and a PSL entry for the parent to the requested domain is what they use to gate that.

**Comment by sleevi** [comment]
Yes, I'm aware of all of this, and perhaps it's worth not using this PR for the meta-discussion here. There's a specific statement that's been made by the requestor that I'd like to better understand, and your attempts have not addressed that. I suspect we're simply not communicating effectively here.

https://github.com/publicsuffix/list/pull/876#issuecomment-611984994
> So in order to provide the students a fairly protected environment, using Cloudflare, we need to fullfill the Cloudflare requirement.

In https://github.com/publicsuffix/list/pull/876#issuecomment-612076102, I'm attempting to understand what is meant by "a fairly protected environment using Cloudflare" here, because that's not something the PSL exists to provide for. 

I'm hoping you'll allow @githeuvel to respond to that, and we can take the conversation about Cloudflare beyond this specific request off the PRs. Thanks.

**Comment by githeuvel** [comment]
@sleevi my apologies, I fear I may have provided too much, or the wrong, details which confuses matters.

Perhaps my reason for this request should have been merely, that I provide free subdomains (to students).

Perhaps by trying to make a stronger case, and giving an actual example, I may have clouded the request?

 - So as a specific example I mentioned CloudFlare which provides [WAF](https://www.cloudflare.com/waf/) for them, but is [not available for them/_subdomains_](https://support.cloudflare.com/hc/en-us/articles/205359838-I-cannot-add-my-domain-to-Cloudflare-)).
- Another example I could have given relates to [cookie scoping of the domain attribute](https://wiki.mozilla.org/Public_Suffix_List/Uses#Cookies).

This request is not specifically targeted at CloudFlare or securing websites. It is/was the first thing they ran into, but is not the _end-goal_
Basically this request is to provide my students a (fairly) real domain experience.

I hope I have answered your questions @sleevi ?

Shall I amend the "Reason for PSL Inclusion" to:

As a private teacher I provide free subdomains to my students. This PR will provide their subdomain a _real_ domain experience (e.g. cookie domain scoping, or services only provided to _real_ domains).

**Comment by sleevi** [comment]
> As a private teacher I provide free subdomains to my students. This PR will provide their subdomain a _real_ domain experience (e.g. cookie domain scoping, or services only provided to _real_ domains).

Right, this is why I want to try to understand. The cookie domain scoping problem is sufficient.

However, the PSL doesn't and isn't meant to ensure services are provided to "real" domains or distinguished as such. When services make assumptions like that (e.g. Let's Encrypt, or, in this case, Cloudflare), those sorts of issues should be taken up with the service provider. The PSL can't and doesn't determine what's a "real" or "real enough" domain, but does exist to help ensure separation of certain legacy browser features ("legacy" as in cookies, so like... widely used, just poorly designed :P)

**Comment by githeuvel** [comment]
@sleevi adjusted the reason accordingly. All green 🚦 now?

**Comment by dnsguru** [comment]
Since the time of this PR a number of other smaller projects that have aspired to use PSL entries work around limits of various sorts have been declined.  We have updated the submission guidelines to reflect this.  [acceptance/validation](https://github.com/publicsuffix/list/wiki/Guidelines#validation-and-non-acceptance-factors)
